src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import exception.DukeErrorException;
+import exception.UnknownCommandException;
+
+/**
+ * Represents parser class to parse user commands
+ */
+public class Parser {
+
+    /**
+     * Method to verify the command is valid or not
+     *
+     * @param commandString user input
+     * @return a {@code Command} object representing user actions

```

Comment:
Maybe you could capitalise the parameter descriptions and punctuate them? Since the conventions page states that we should have "Punctuation behind each parameter description". You could also refer to the given example as reference 👍 .